base/number.jl

```diff
@@ -63,4 +63,6 @@ zero{T<:Number}(::Type{T}) = convert(T,0)
 one(x::Number)  = oftype(x,1)
 one{T<:Number}(::Type{T}) = convert(T,1)
 
+_default_type(::Type{Number}) = Int
```

Comment:
What's wrong with leaving it as the default (`Number`) rather than returning a concrete type?